test/test_retriever.py

```diff
@@ -681,13 +688,20 @@ def test_reset_retriever(tmpdir):
     pwd_name = os.getcwd()
     workdir = tmpdir.mkdtemp()
     workdir.chdir()
-    dataset = random.choice(rt.dataset_names())
+    offline_datasets = rt.dataset_names()['offline']
+    offline_datasets = [dataset for dataset in offline_datasets if not dataset.startswith('test-')]
+    if len(offline_datasets) == 0:
```

Comment:
`if not offline_datasets` should be the same as `if len(offline_datasets) == 0:`